### ISSUE TITLE
Error in re-opening a saved post with a plugin-based shortcode

### ISSUE THREAD

**Issue by barrycollard** [WordPress/gutenberg#7833]
When using both the shortcode plugin '**List Category Posts**' or the older '**Display Posts Shortcode**' plugin, the page accepts the shortcode and provides a working preview and published page. When the page is re-opened to be edited, the page is redirecting to a non-relevant post and won't allow the Gutenberg editor to open. The matter can be resolved by opening the page within a classic editor, removing the offending line, and saving. The page can then be opened using the Gutenberg editor.

**To Reproduce**
1. Open a new page and create a new shortcodes block
2. With the 'List Category Posts' plugin installed, type in default shortcode like [catlist "name=post_type="post"]
2. Click on save or publish
3. Go back to the 'Pages > All Pages' section within the Dashboard and open the new page
4. The error should be confirmed when the page opens to an incorrect post or page

**Expected behavior**
The newly created Gutenberg-based editing page should allow the page to re-open for editing purposes.

**Desktop (please complete the following information):**
 - OS: Windows 7
 - Browser: Tried in Firefox Developer Editon
 - Version [62]

**Additional context**
- Created using Gutenberg 3.2.0.
- Oddly, the issue doesn't appear to be a problem when using a simple gallery such as [gallery ids="2456"] (NB: the image id is just an example)

**Comment by danielbachhuber** [comment]
Do you think #7468 could be related?

**Comment by easilyamused** [comment]
I can look into this more, but our pages do use shortcodes to list posts They are our custom shortcodes.

Example page:
https://www.freshfarmscbd.com/

**Comment by barrycollard** [comment]
After trying to replicate the issue this morning on a different site for which I'm the admin, I found that the '**List Category Posts**' plugin was working but the '**Display Posts Shortcode**' plugin still returns an error. I ran the same test again this morning on my original site and this now also appears to be returning the same response, as just described.

'**List Category Posts**' did issue an update last night to their plugin so I'm not sure if this is to do with the error now appearing to be fixed. As '**Display Posts Shortcode**' is now out-of-date by 10 months, I would think that it's _not_ worth chasing up with the plugin's owner.
